components/load_from_hf_hub/fondant_component.yaml

```diff
@@ -3,20 +3,18 @@ description: Component that loads a dataset from the hub
 image: ghcr.io/ml6team/load_from_hf_hub:latest
 
 produces:
-  images:
-    fields:
-      data:
-        type: binary
-      width:
-        type: int16
-      height:
-        type: int16
-  captions:
-    fields:
-      data:
-        type: string
+  # fill here
 
 args:
   dataset_name:
     description: Name of dataset on the hub
     type: str
+  column_name_mapping:
+    description: column to map the original column names of the input dataset to
+    type: dict
```

Comment:
are we assuming that the dataset only has these two columns? What are we doing with multiple image columns etc?

Reply:
We should probably turn the `image_column_name` into an array?

Reply:
yes it should be a `list` since we might have more than one image, i'll adjust it